packages/trader/src/Assets/Trading/Categories/Description/multiplier-trade-description.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Localize } from '@deriv/translations';
+import { Text } from '@deriv/components';
+
+const MultiplierTradeDescription = ({
+    is_multiplier_fx,
+    onClick,
+}: {
+    is_multiplier_fx?: boolean;
+    onClick: React.MouseEventHandler<HTMLSpanElement>;
+}) => {
+    const content = [
+        <Localize
+            i18n_default_text='Use multipliers to leverage your potential returns. Predict if the asset price will move upward (bullish) or downward (bearish). We’ll charge a commission when you open a multipliers trade.'
+            key='1'
```

Comment:
we don't put keys in the Localize component. Lets keep it consistent.

Reply:
You can't store <Localize /> in array without key. Btw keys was also used in Package 6 in same situation 
<img width="905" alt="Screenshot 2023-10-09 at 09 05 52" src="https://github.com/binary-com/deriv-app/assets/121025168/ea904e08-eea2-4773-9925-ec02649cf35c">

Reply:
yeah this case is ok but u put keys in places where its just one localize component no?